### ISSUE TITLE
load_image_dataset

### ISSUE THREAD

**Issue by X-YI** [jhfjhfj1/autokeras#47]
autokeras.classifier.load_image_dataset failed silently

x_train, y_train = load_image_dataset(csv_file_path="***_train.csv", images_path=".")
print(x_train.shape)
print(y_train.shape)

***_train.csv contains 100k lines of image path and labels

I try to do load_image_dataset, and wish to get x_train.shape. However, it doesn't print any value.
so is the load_image_dataset not able to handle 100k data size?

**Comment by jhfjhfj1** [comment]
Thank you for your interest.

I think it can.
Did you try the load_raw_data example?
@X-YI

**Comment by X-YI** [comment]
sorry currently I have no time to confirm these so I’d like to close it temporarily
